_data/projects/home-assistant.io.yml

```
@@ -9,7 +9,7 @@ tags:
 - hass
 - hassio
 upforgrabs:
-  name: hacktoberfest
+  name: help wanted
   link: https://github.com/home-assistant/home-assistant.io/labels/hacktoberfest
```

This also needs to be in sync with the label change:

```suggestion
  link: https://github.com/home-assistant/home-assistant.io/labels/Help%20wanted
```